test/e2e/snaps/test-snap-bip-44.spec.js

```diff
@@ -53,6 +53,13 @@ describe('Test Snap bip-44', function () {
           tag: 'button',
         });
         await driver.waitForSelector({ text: 'Install' });
+
+        try {
+          await driver.clickElement('[data-testid="snap-install-scroll"]');
+        } catch (_) {
+          console.log('Missing scroll');
+        }
```

Comment:
Can we do a util function for this, since we're doing this in multiple places?

Something like

```ts
function safeClickElement(driver, selector);
```

which just does the try-catch. It should have some documentation around why we need this.

Reply:
Added `clickElementSafe` to driver.js and implemented it in the three offending tests.